config/config.exs

```diff
@@ -49,12 +49,25 @@ config :phoenix, :json_library, Jason
 # Use Jason for JSON parsing in ExAws
 config :ex_aws, json_codec: Jason
 
-config :screenplay, ScreenplayWeb.AuthManager, issuer: "screenplay"
+# 12 hours in seconds
+max_session_time = 12 * 60 * 60
+
+config :screenplay, ScreenplayWeb.AuthManager,
+  issuer: "screenplay",
+  max_session_time: max_session_time,
+  # 30 minutes
+  idle_time: 30 * 60
```

Comment:
**question:** Is this also a "max" value? Should we call it `max_idle_time`?

Reply:
Sure, that seems more appropriate here.